docs/installation/sources/sources_mac.rst

```diff
@@ -118,6 +118,10 @@ Dependencies
 * :ref:`asiotinyxml2_sm`
 * :ref:`openssl_sm`
 
+.. seealso::
+  For further information about this Fast DDS version dependencies and other products related versions, please check
+  the Fast DDS :ref:`related_versions` section.
```

Comment:
If the suggestion  about the name of the `related_versions` section is applied, please change also here. Moreover, if we change the name we can refer directly to the specific section inside the page, i.e. `Build time dependencies`